locales/en-US/app.ftl

```diff
@@ -75,6 +75,7 @@ emailOptInDialogErrorTitle = Oh no!
 ## News feed updates.
 latestUpdatesTitle = Latest updates
 showMoreNewsTitle = Show Past News
+englishArticleLink = Link to English article
```

Comment:
I would add a comment explaining "Link" is a noun, just to avoid any possible confusion.